### DISCUSSION TITLE
Lesson 7: Assertion error

### DISCUSSION THREAD

**Discussion by andremiracle** [smartcontractkit/full-blockchain-solidity-course-py#551] [Q&A]
When I try to "brownie test --network mainnet-fork" from around (6:28:55) in the video, I get an assertion error. I don't know the cause of this, can someone help me. Thanks.
This is my test_lottery.py code
![Screenshot (35)](https://user-images.githubusercontent.com/39734097/145774631-b962d622-a29d-48fb-8191-36d1f52aecba.png)
This is my yaml file
![Screenshot (36)](https://user-images.githubusercontent.com/39734097/145774644-e034122d-45e4-4ca2-ae58-a0d9b285e4df.png)
this is the error
![Screenshot (37)](https://user-images.githubusercontent.com/39734097/145774649-865580f7-c1f7-4783-8e17-87c32aa0f920.png)

**Comment by cromewar** [comment]
Hello @andremiracle as you can see the reasons is due to the amount you are sending, the entrance fee is $50 which is this amount on eth:

![image](https://user-images.githubusercontent.com/20199180/145819100-fbdc75c1-c216-4f1a-af1b-f4cb01f1f2f0.png)

So, the assertion is failing because the entrance fee is not higher than 0.18 eth. Just change those values and you'll solve this.

best regards, Cromewar.
